docs/source/markdown/podman-system-prune.1.md

```diff
@@ -18,6 +18,21 @@ By default, volumes are not removed to prevent important data from being deleted
 
 Recursively remove all unused pod, container, image and volume data (Maximum 50 iterations.)
 
+#### **--filter**=*filters*
+
+Provide filter values.
+
+The --filter flag format is of “key=value”. If there is more than one filter, then pass multiple flags (e.g., --filter "foo=bar" --filter "bif=baz")
+
+Supported filters:
+
+- `until` (_timestamp_) - only remove containers and images created before given timestamp
+- `label` (label=_key_, label=_key=value_, label!=_key_, or label!=_key=value_) - only remove containers and images, with (or without, in case label!=... is used) the specified labels.
+
+The until filter can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the daemon machine’s time.
```

Comment:
Same

Reply:
Thanks @ashley-cui
Fixed.

Reply:
I think we need to convene a kangaroo court and fine @rhatdan $25 to be donated to a charity of his choice for inserting daemon there!

Reply:
More of a cut and paste, and missed removing it.